SiG/laser-util-sig.py

```diff
@@ -44,6 +45,7 @@ def __init__(self):
         parser.add_argument("--raman-delay-ms",      type=int,            help="set laser warm-up delay in Raman Mode (~ms)")
         parser.add_argument("--raman-mode",          type=str,            help="dis/enable raman mode (links firing to integration) (bool)")
         parser.add_argument("--selected-adc",        type=int,            help="set selected adc")
+        parser.add_argument("--adc-raw",         type=int,            help="get raw ADC values")
```

Comment:
You might mention in "help" that the integral argument is the ADC to be read, and is typically 0 or 1.  Which ADC do you read to get the laser temperature?